xcresult/tests/data/test4.junit.xml

```diff
@@ -1,919 +1,1115 @@
+    </testsuite>
+    <testsuite name="UpcomingMoviesTests.MovieDetailTitleRenderContentTests" tests="4" disabled="0" errors="0" failures="0">
+        <testcase name="testReleaseDate()" time="0.000" id="8f088813-b061-5a0d-9295-823a187d62d7">
         </testcase>
-        <testcase name="testTitle()" classname="MovieDetailTitleRenderContentTests" time="0.000" id="a359ccf0-7d96-5c6f-8a9a-063e7d1e8646">
+        <testcase name="testTitle()" time="0.000" id="b2e56333-3dfb-54ea-b784-d5f7bce0ad4d">
         </testcase>
-        <testcase name="testVoteAverage()" classname="MovieDetailTitleRenderContentTests" time="0.000" id="f70d5d9a-ae1c-5510-a4b0-e3b1616c2c06">
+        <testcase name="testVoteAverage()" time="0.000" id="d2b44311-70c1-5a17-b604-5f72993a5005">
         </testcase>
-        <testcase name="testGenreIds()" classname="MovieDetailTitleRenderContentTests" time="0.000" id="39d5c9fa-deb4-5571-b917-43729e3a794a">
+        <testcase name="testGenreIds()" time="0.000" id="4ec5d8e5-6aac-5f77-85a6-bc067898faeb">
         </testcase>
-        <testcase name="testGetFavoriteListCalled()" classname="FavoritesSavedMoviesInteractorTests" time="0.001" id="3d9b7d90-d52c-503b-96d6-7ebfd2c03fe3">
+    </testsuite>
+    <testsuite name="UpcomingMoviesTests.FavoritesSavedMoviesInteractorTests" tests="1" disabled="0" errors="0" failures="0">
+        <testcase name="testGetFavoriteListCalled()" time="0.001" id="ebca35ea-ecbb-5857-89f0-266c818336cf">
         </testcase>
-        <testcase name="testBuild()" classname="RecommendedMoviesCoordinatorTests" time="0.004" id="4a4180ab-cf44-58d8-8126-ca091a3ab348">
+    </testsuite>
+    <testsuite name="UpcomingMoviesTests.RecommendedMoviesCoordinatorTests" tests="1" disabled="0" errors="0" failures="0">
+        <testcase name="testBuild()" time="0.004" id="d60fff18-91e9-5f6c-9bfc-3644dcd6ebfa">
         </testcase>
-        <testcase name="testBuild()" classname="MovieCreditsCoordinatorTests" time="0.003" id="eb8b05de-cb35-5c43-bc4a-0ef03c175bd3">
+    </testsuite>
+    <testsuite name="UpcomingMoviesTests.MovieCreditsCoordinatorTests" tests="1" disabled="0" errors="0" failures="0">
+        <testcase name="testBuild()" time="0.003" id="b830e5d1-35d3-5d15-b198-714744abfb90">
         </testcase>
-        <testcase name="testInitWithMovieSearch()" classname="MovieSearchModelTests" time="0.011" id="6a0ee60a-0a5e-5050-83ac-3eb22a6cb142" file="/Users/work/src/iOS-example/UpcomingMoviesTests/MovieSearchModelTests.swift">
-            <failure message="XCTAssertEqual failed: (&quot;12345&quot;) is not equal to (&quot;12346&quot;)"/>
+    </testsuite>
+    <testsuite name="UpcomingMoviesTests.MovieSearchModelTests" tests="1" disabled="0" errors="0" failures="1">
+        <testcase name="testInitWithMovieSearch()" time="0.011" id="a4988fb7-1943-5c3b-8ac5-1eccca86cc46">
+            <failure message="MovieSearchModelTests.swift:21: XCTAssertEqual failed: (&quot;12345&quot;) is not equal to (&quot;12346&quot;)"/>
```

Comment:
I forgot about this `file` attribute... The new `xcrun xcresulttool` output does not include any files now. It does include "source code references" in the output of the `xcrun xcresulttool get test-results test-details` though. Let's discuss on Slack or on a call